src/components/task-plan/select-topics.cjsx

```diff
@@ -0,0 +1,136 @@
+React = require 'react'
+_ = require 'underscore'
+BS = require 'react-bootstrap'
+Dialog = require '../dialog'
+{TocStore, TocActions} = require '../../flux/toc'
+{TaskPlanStore, TaskPlanActions} = require '../../flux/task-plan'
+
+SectionTopic = React.createClass
+  displayName: 'SectionTopic'
+
+  propTypes:
+    planId: React.PropTypes.any.isRequired
+    section: React.PropTypes.object.isRequired
+    active: React.PropTypes.bool
+
+  render: ->
+    classes = ['section']
+    classes.push('selected') if @props.active
+    isChecked = 'checked' if @props.active
+    classes = classes.join(' ')
+
+    <div key={@props.section.id} className={classes} onClick={@toggleSection}>
+      <span className="-section-checkbox">
```

Comment:
Likely no, just added that in there in case Fred/Dak wanted to use it to style a bit.  Should I remove it?